src/setStateIndicator/randomNum.js

```diff
@@ -0,0 +1,15 @@
+export const getRandomNumber = (max) => {
+  return Math.floor(Math.random() * max);
+};
+
+export const setStateDependingRandomNum = (maxNum, setIndicator) => {
+  let timerId = setInterval(() => {
+    let randomNumber = getRandomNumber(maxNum);
+    console.log("randomNum", randomNumber);
+    if (randomNumber === 1) {
+      setIndicator("sick");
+      clearInterval(timerId);
+      return;
+    }
+  }, 5000);
```

Comment:
let's indicate in a constant what this timer means

Reply:
done